lib/rules/no-await-sync-events.ts

```diff
@@ -10,13 +10,23 @@ import {
 } from '../node-utils';
 
 const USER_EVENT_ASYNC_EXCEPTIONS: string[] = ['type', 'keyboard'];
-const VALID_EVENT_MODULES = ['fire-event', 'user-event'] as const;
+const FIRE_EVENT_OPTION = 'fire-event' as const;
+const USER_EVENT_OPTION = 'user-event' as const;
+const VALID_EVENT_MODULES = [FIRE_EVENT_OPTION, USER_EVENT_OPTION];
+const DEFAULT_EVENT_MODULES = [FIRE_EVENT_OPTION];
 
 export const RULE_NAME = 'no-await-sync-events';
 export type MessageIds = 'noAwaitSyncEvents';
-type Options = [
-	{ eventModules?: readonly (typeof VALID_EVENT_MODULES)[number][] }
-];
+
+type ValidEventModules = (typeof VALID_EVENT_MODULES)[number];
+type EventModulesOptions = ReadonlyArray<ValidEventModules>;
+type Options = [{ eventModules?: EventModulesOptions }];
+
+function getEnabledEventModules(
+	eventModulesOption?: EventModulesOptions | undefined
+): ReadonlyArray<ValidEventModules> {
+	return eventModulesOption ?? DEFAULT_EVENT_MODULES;
+}
```

Comment:
This is basically just setting a default value now, so we can inline this in the `options` destructuring

```suggestion
```